atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ResultsExtractor.java

```diff
@@ -78,15 +78,15 @@ public ResultsExtractor(MetricsManager metricsManager) {
             ColumnSelection selection,
             byte[] endExclusive) {
         byte[] lastRow = extractResults(colsByKey, startTs, selection);
-        SortedMap<byte[], SortedMap<byte[], T>> resultsByRow = Cells.breakCellsUpByRow(asMap());
+        NavigableMap<byte[], NavigableMap<byte[], T>> resultsByRow = Cells.breakCellsUpByRow(asMap());
         return getRowResults(endExclusive, lastRow, resultsByRow);
     }
 
     public static <T> TokenBackedBasicResultsPage<RowResult<T>, byte[]> getRowResults(
             byte[] endExclusive,
             byte[] lastRow,
-            SortedMap<byte[], SortedMap<byte[], T>> resultsByRow) {
-        SortedMap<byte[], RowResult<T>> ret = RowResults.viewOfSortedMap(resultsByRow);
+            NavigableMap<byte[], NavigableMap<byte[], T>> resultsByRow) {
```

Comment:
This one doesn't seem to have external users, so should be fine

Reply:
Still added to changelog.